SeleniumTrainingCenter/BaseTestClass.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace SeleniumTrainingCenter
+{
+    [TestClass]
+    public class BaseTestClass
+    {
+        protected const string _chromeDriverPath = @"C:\Users\AndriusBogda\Desktop\chromedriver_win32 (1)";
```

Comment:
Please, use https://www.nuget.org/packages/Selenium.WebDriver.ChromeDriver/ within your project. It will allow you don't set the path to webdriver directly. Just use "new ChromeDriver();" - it will find necessary .exe in bin folder.